.github/workflows/label_merge_conflicts.yml

```diff
@@ -0,0 +1,23 @@
+name: Manage labels
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should this just be limited to a push to master? It's probably fine, but if in the future we switched to having multiple branches, it might cause problems.

Reply:
There are other workflows using these restrictions, so it's probably fine.